edx-platform/pearson-theme/lms/templates/header/navbar-not-authenticated.html

```diff
@@ -13,25 +13,31 @@
 
 <%
   mktg_site_enabled = static.get_value('ENABLE_MKTG_SITE', settings.FEATURES.get('ENABLE_MKTG_SITE', False))
-  courses_are_browsable = settings.FEATURES.get('COURSES_ARE_BROWSABLE')
+  courses_are_browsable = static.get_value('COURSES_ARE_BROWSABLE', settings.FEATURES.get('COURSES_ARE_BROWSABLE', False))
   allows_login = not settings.FEATURES['DISABLE_LOGIN_BUTTON'] and not combined_login_and_register
   can_discover_courses = settings.FEATURES.get('ENABLE_COURSE_DISCOVERY')
   allow_public_account_creation = static.get_value('ALLOW_PUBLIC_ACCOUNT_CREATION', settings.FEATURES.get('ALLOW_PUBLIC_ACCOUNT_CREATION'))
+  how_it_works_link_enable = static.get_value('HOW_IT_WORKS_LINK_ENABLE', settings.FEATURES.get('HOW_IT_WORKS_LINK_ENABLE', False))
+  schools_link_enable = static.get_value('SCHOOLS_LINK_ENABLE', settings.FEATURES.get('SCHOOLS_LINK_ENABLE', False))
 %>
 <nav class="nav-links" aria-label=${_("Supplemental Links")}>
   <div class="main">
   % if mktg_site_enabled:
-    <div class="mobile-nav-item hidden-mobile nav-item">
-      <a href="${marketing_link('HOW_IT_WORKS')}">${_("How it Works")}</a>
-    </div>
+    % if how_it_works_link_enable:
+      <div class="mobile-nav-item hidden-mobile nav-item">
+        <a href="${marketing_link('HOW_IT_WORKS')}">${_("How it Works")}</a>
```

Comment:
Is marketing_link site-aware? @luismorenolopera

Reply:
yes @Squirrel18